src/Expeditions/Application/Handler/CreateExpeditionCommandHandler.php

```diff
@@ -0,0 +1,41 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Expeditions\Application\Handler;
+
+use App\DomainModel\Repository\ExpeditionRepository;
+use App\DomainModel\Repository\MarsScientistRepository;
+use App\Entity\Expedition;
+use App\Expeditions\Application\Command\CreateExpeditionCommand;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+use Symfony\Component\Security\Core\Security;
+
+class CreateExpeditionCommandHandler implements MessageHandlerInterface
+{
+    private Security $security;
+    private ExpeditionRepository $expeditionRepository;
+    private MarsScientistRepository $marsScientistRepository;
+
+    public function __construct(Security $security, ExpeditionRepository $expeditionRepository, MarsScientistRepository $marsScientistRepository)
```

Comment:
Taki jeden trick Ci pokaże z PHP 8.0 ;p

```suggestion
    public function __construct(
        private Security $security,
        private ExpeditionRepository $expeditionRepository,
        private MarsScientistRepository $marsScientistRepository
    )
```

I po tym możesz wywalić deklarację zmiennych i ich inicjalizacje - PHP robi to za ciebie ;p